source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Automatically Test and Deploy GitHub Changes to Pantheon from an Existing Jenkins Server
+description: Configure an existing Jenkins server to automatically test and deploy code changes to Pantheon when pushed to GitHub.
+tags: [siteintegrations]
+type: guide
+permalink: docs/guides/:basename/
+date: 6/20/2017
+---
+
+[Jenkins](https://jenkins.io) is an open source Continuous Integration (CI) server which can be used to build, test, and deploy code on any Drupal and WordPress website on Pantheon. Unlike hosted services such as [CircleCI](https://circleci.com/), it is a Java application installed and run on a server, and requires regular upkeep and maintenance. However, it is very customizable and can even run non-CI related tasks, such as calling periodic cron jobs.
+
+In this guide, we'll configure your existing Jenkins server to automatically test and deploy Drupal 8 to a Multidev environment on Pantheon following a code push to your GitHub repository. Jenkins will run tests against both the pull request and Pantheon's master branch and display the results.
```

Comment:
I revised this a bit, but left the last sentence the same because it confused me. 

Where are the results displayed? Saying that Jenkins will run tests against the pull request and master and display the results makes me think that there will be some kind of comparison between the results, but I don't think that's the case. 

Pull request may not be the best term to use here, since tests are run against the tip of a branch. Suggest something like code change or commit, whatever triggers a build.